runtime/src/main/java/org/corfudb/runtime/collections/CorfuStore.java

```diff
@@ -201,24 +202,46 @@ public Query query(@Nonnull final String namespace) {
     /**
      * Subscribe to a specific a table in a namespace or the entire namespace.
```

Comment:
existing typo: Subscribe to a specific table in a namespace or to all tables in a namespace

Reply:
Done.